change/@fluentui-react-shared-contexts-cb55fa3a-95bc-492f-aa57-1a7616941102.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: Implement AvatarContext",
+  "packageName": "@fluentui/react-shared-contexts",
+  "email": "lingfangao@hotmail.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
This can be removed

Reply:
Removed in https://github.com/microsoft/fluentui/pull/24807/commits/ef70cc85d7564264a5c0978848ed470996deda60